docs/console/company-configuration/settings.md

```diff
@@ -0,0 +1,22 @@
+---
+id: settings
+title: Company Settings
+sidebar_label: Company Settings
+---
+
+The **Settings** section allows you to view and define various configurations and settings related to your Company. These can include general settings, feature preview options, advanced settings and so on. 
+
+### Rules
+
+The **Rules** tab allows the user to customize rules defined for the Company and its Projects. Rules can pertain to different domains, such as configuration, runtime or deployment management.
```

Comment:
We shall document what exists now, we don't have runtime or deployment rules and reading this can confuse users

Reply:
I agree, that's the comment I was waiting for :P